src/components/ConfirmModal.js

```diff
@@ -74,4 +80,11 @@ ConfirmModal.defaultProps = {
   with_resend_link: true
 };
 
-export default i18n(ConfirmModal);
+const mapStateToProps = state => {
+  return {
+    formEnabled: isValid('modal-form')(state) && !isSubmitting('modal-form')(state)
+  };
+};
+
+const ConfrimModalContainer = connect(mapStateToProps, null)(ConfirmModal);
+export default i18n(ConfrimModalContainer);
```

Comment:
According to the folder structure that applies to modals, until we build a new one, the marked code should be saved in a separate file in `containers/`(named after the component)